test/parallel/test-util-deprecate-invalid-code.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const common = require('../common');
+const util = require('util');
+
+[1, true, false, null, {}].forEach(notString => {
```

Comment:
Caught by linter: this needs parens around `notString` to be `(notString)`.

Reply:
@akaila You can run linter just for Javascript files using `make lint-js`

Reply:
Ok trying now.